config/initializers/assets.rb

```diff
@@ -1,7 +1,7 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.version = '1.0'
+Rails.application.config.assets.version = 'v1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.